topcoffea/params/xsec.yml

```diff
@@ -91,7 +91,7 @@ ZZTo2Q2Nu               : 4.04
 ZZTo4L                  : 1.256
 
 ggToZHToHToWWTo2L2Nu    : 0.00275
-qqToZHToZTo2L           : 0.0018639
+ggToZHToZTo2L           : 0.0018639
```

Comment:
Was this change intentional? Was it a typo in the previous version? Do you have a ref for the xsec?

Reply:
definitely not intentional. Not sure when this happened. Maybe a typo in the previous version that has since been updated and I pushed the old?